src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/DocumentManagementService.java

```diff
@@ -205,7 +205,7 @@ public UploadedDocumentType addDocumentToDocumentField(DocumentInfo documentInfo
      */
     public String displayDocNameTypeSizeLink(UploadedDocumentType document, String authToken) {
         if (document == null) {
-            return "";
+            return null;
```

Comment:
assuming this null is handled correctly when returned

Reply:
Yeah the consumer has changed behaviour so this needs to change to reflect it, its only used in these two places